### DISCUSSION TITLE
Can't get any of the "How Can I Get It" approaches to work

### DISCUSSION THREAD

**Discussion by tomRedox** [Dotnet-Boxed/Templates#177] [Q&A]
I can't get any of the approaches here https://github.com/ASP-NET-Core-Boilerplate/Templates#how-can-i-get-it to actually work.

**Nuget**:  Following through the steps, if I do step 4 without adding any extra options it creates a project with almost nothing in it and broken reference to Microsoft.AspNetCore.All

![image](https://user-images.githubusercontent.com/11335350/28430075-1180901a-6d77-11e7-964f-2c97231ac35b.png)

**VSIX - Visual Studio Gallery**:  This installs OK and I get the templates option in the new project dialog in VS, but this appears to be pointing to an out of date version of the template (Version 6.0.0.226 Last updated 06/04/2017, 17:15:07 according to https://marketplace.visualstudio.com/items?itemName=RehanSaeed.ASPNETMVCBoilerplate).  When I use it to create a project it has the missing pfx file issue from a case you closed last month.

 **VSIX - My-Get Feed**:  That seems to add the extension fine, but I've got no idea how to then use it to create a project, there's no ASP Core Boilerplate template in the new project dialog as far as I can see.

**Git Clone** - this clones the repo you use to create the VSIXs I think, so it's not a way to actually build a new project based on the boilerplate as far as I can tell?  If so, it probably shouldn't be in this list.

So, I'm stuck!

**Comment by RehanSaeed** [comment]
# NuGet
Looks like you have preview 2 installed. That will use the new preview 2 version of `dotnet new`, while the template is version 1.1.0. Put a global.json file in the folder so you can tell `dotnet` you want the 1.1.0 version of it. Unfortunately, there is not much I can do about this one, you just need to know about `global.json`. I'll add a note to the ReadMe.md.

# VSIX
This is totally my fault. I've been snowed under and haven't had time to release the fix which is in the master waiting for some testing on my part. This Saturday, it's my goal to get this out.

# Git Clone
Yes, this requires some specialist knowledge but I try to make it easier with the HOW TO BUILD.md file.

**Comment by RehanSaeed** [comment]
Have pushed updates to both packages. Let me know if you have any issues.
